src/sonic-build-hooks/scripts/post_run_buildinfo

```diff
@@ -1,12 +1,22 @@
 #!/bin/bash
+#set -x
+
+IMAGENAME=$1
 
 . /usr/local/share/buildinfo/scripts/buildinfo_base.sh
 
 # Collect the version files
 collect_version_files $POST_VERSION_PATH
 
-[ -d $BUILD_VERSION_PATH ] && [ ! -z "$(ls -A $BUILD_VERSION_PATH)" ] && cp -rf $BUILD_VERSION_PATH/* $POST_VERSION_PATH
-rm -rf $BUILD_VERSION_PATH/*
+	
+# After the docker build, save the cache contents as tar file and send them to host through log file.
+# Cache contents are encoded as base64 format to avoid the log parsing in the host.
+tar -C ${PKG_CACHE_PATH} --exclude=cache.tgz -zcvf ${PKG_CACHE_FILE_NAME} .
+#set +x
+if [[ ! ${IMAGENAME} =~ host-image ]]; then
+	sleep 1;echo  -e "\n_VCSTART_"; (cat ${PKG_CACHE_FILE_NAME} | base64); echo -e "_VCEND_\n";sleep 1
```

Comment:
Can we use the scp command or the other RPC APIs to copy the content to the slave container? It is a little workaround to copy content by log.